### ISSUE TITLE
algo pyrinhash is a secret

### ISSUE THREAD

**Issue by PSLLSP** [rigelminer/rigel#116]
New algo `pyrinhash` is not documented in `--help` output:
```
root@HIVEOS:/hive/miners/rigel/1.11.0# ./rigel -V
rigel 1.11.0
```
```
root@HIVEOS:/hive/miners/rigel/1.11.0# ./rigel --help | grep -i pyrin
root@HIVEOS:/hive/miners/rigel/1.11.0#
```

**Comment by rigelminer** [comment]
Oops, it went out of sync with the github readme. Will fix in the next version. Thanks for reporting.

**Comment by rigelminer** [comment]
Fixed in 1.11.1
